content/configuration/_index.md

```diff
@@ -41,21 +41,47 @@ The configuration file contains,
   # about the best practices for the created resource
   recommendations: true
   
-  # Channels configuration
+  # Communcation mediums configuration
   communications:
+    # Settings for Slack
     slack:
+      enabled: false
       channel: 'SLACK_CHANNEL'
       token: 'SLACK_API_TOKEN'
-
+  
+    # Settings for Mattermost
+    mattermost:
+      enabled: false
+      url: 'MATTERMOST_SERVER_URL'                # e.g https://example.com:9243
+      token: 'MATTERMOST_TOKEN'
+      team: 'MATTERMOST_TEAM'
+      channel: 'MATTERMOST_CHANNEL'

```

Comment:
@mugdha-adhav can you please add one liner description for these fields? Like we have for `resources`

Reply:
Made required changes.